lib/widgets/breez_avatar.dart

```diff
@@ -43,7 +45,12 @@ class BreezAvatar extends StatelessWidget {
   Widget build(BuildContext context) {
     Color avatarBgColor =
         this.backgroundColor ?? theme.sessionAvatarBackgroundColor;
+
     if (avatarURL != null && avatarURL.isNotEmpty) {
+      if (avatarURL.startsWith("/data")) {
```

Comment:
We shouldn't assume "/data" prefix on file urls.
A simple solution is to check first for "breez://profile_image" prefix and if not assume it is file.

Reply:
I believe uploaded image takes precedence over "breez://profile_image".
Also, we can't assume it's file, if it does not start with "breez://profile_image" because there are several other cases.
![image](https://user-images.githubusercontent.com/4012752/75726812-608f4880-5cf4-11ea-8765-fe723036ab4d.png)

How about
```suggestion
      if (avatarURL.contains("/image_cropper")) {
```

Reply:
I don't think it is robust enough.
We can check by the following sequence:
* "breez://profile_image" prefix.
* "src/icon/vendors"
* check for http url (Uri.parse and make sure the scheme starts with http).
* fallback to file.

Reply:
[Done](https://github.com/breez/breezmobile/pull/283/commits/9d49f38a42ccdaa6615cc70699655d7295bbb639)